definitions/tools/somatic_concordance_graph.cwl

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Generate somatic concordance report graph"
+baseCommand: ["/usr/local/bin/Rscript", "somatic_concordance_graph.R"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "registry.gsc.wustl.edu/mgi-cle/somatic-concordance-report:v1"
```

Comment:
Is there anything in this image that prevents it from being released publicly?

Reply:
Not really. This local registry has several pre-installed R libraries and the R script so we can manually run the script. This is very specific for cle group purpose and probably is not much useful outside of cle.

Reply:
Yeah, I don't know who might want to use this tool or workflow.  In the absence of reasons to do otherwise [like licensing restrictions or protected information], I have a bias towards keeping the contents of this repo functional for external users.